### PR TITLE
contact form

### PR DESCRIPTION
**Pull request by smanousopoulos**

### CODE REVIEW

File: ckanext/publicamundi/themes/geodata/plugin.py [hunk 1]
```diff
@@ -187,8 +199,43 @@ class GeodataThemePlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IRoutes, inherit=True)
     plugins.implements(plugins.IPackageController, inherit=True)
 
-    # ITemplateHelpers
-    
+    def init_contact_captcha(self):
+
+        from wheezy.captcha.image import captcha
```
Comment (kalxas) [inline ckanext/publicamundi/themes/geodata/plugin.py]: Is this Debian specific code?

Comment (smanousopoulos) [inline ckanext/publicamundi/themes/geodata/plugin.py]: this is a test code section i added while trying to implement a captcha in the contact form, but actually never executes. Will remove in next commit!

Comment (kalxas) [inline ckanext/publicamundi/themes/geodata/plugin.py]: ok, thanks

Comment (drmalex07) [inline ckanext/publicamundi/themes/geodata/plugin.py]: No, actually `wheezy` is the name of the library that generates captchas

**Pull request merged**